### ISSUE TITLE
[java]UncommentedEmptyMethodBody can not detect empty constructor

### ISSUE THREAD

**Issue by wuchiuwong** [pmd/pmd#2443]
**Affects PMD Version:** 
6.22.0

**Rule:**
UncommentedEmptyMethodBody

**Description:**
UncommentedEmptyMethodBody can not detect empty constructor
This rule is implemented through xpath search:
```
//MethodDeclaration/Block[count(BlockStatement) = 0 and @containsComment = 'false']
```
AST type of constructor is <code>ConstructorDeclaration</code>, so constructor is ignored.

**Code Sample demonstrating the issue:**
```
public Messages() {
}
```

**Expected outcome:**
false-negative

**Running PMD through:**
CLI

**Comment by oowekyala** [comment]
There is another rule called [UncommentedEmptyConstructor](https://pmd.github.io/latest/pmd_rules_java_documentation.html#uncommentedemptyconstructor). Maybe we should merge those two?

**Comment by jsotuyod** [comment]
I support merging the rules, definitely.

**Comment by oowekyala** [comment]
#2902 now tracks this, for clarity
